step-cli/step-cli-launcher/src/main/java/step/cli/StepConsole.java

```diff
@@ -330,9 +330,9 @@ public static class ApExecuteCommand extends AbstractApCommand implements Callab
             protected Integer numberOfThreads;
 
             @Option(names = {"--reportType"}, description = "The type of execution report to be generated and stored locally")
-            protected String reportType;
+            protected List<String> reportType;
```

Comment:
We should list the known report type in the description

Reply:
I've added the the only supported report type (junit) to the description